README.md

```diff
@@ -12,101 +12,17 @@
 
 ---
 
-This monorepo contains code for open-source Code PushUp NPM packages:
-
-- [@code-pushup/cli](./packages/cli) - **CLI** for **collecting** audit results and **uploading** report to portal
-- [@code-pushup/core](./packages/core) - implementation of **core business logic** (useful for custom integrations)
-- [@code-pushup/models](./packages/models/) - **schemas and types** for data models (useful for custom plugins or other integrations)
-- [@code-pushup/utils](./packages/utils/) - various **utilities** (useful for custom plugins or other integrations)
-- plugins:
-  - [@code-pushup/eslint-plugin](./packages/plugin-eslint/) - static analysis using **ESLint** rules
-
-## Contributing
-
-### Setup
-
-Prerequisites:
-
-- Node.js installed (LTS version)
-
-Make sure to install dependencies:
-
-```sh
-npm install
-```
-
-### Development
-
-Refer to docs on [how to run tasks in Nx](https://nx.dev/core-features/run-tasks).
-
-Some examples:
-
-```sh
-# visualize project graph
-npx nx graph
-
-# run unit tests for all projects
-npx nx run-many -t unit-test
-
-# run integration tests for all projects
-npx nx run-many -t integration-test
-
-# run E2E tests for CLI
-npx nx e2e cli-e2e
-
-# build CLI along with packages it depends on
-npx nx build cli
+|                              📊 Getting Started                              |                           🌐 Portal Integration                            |                          🛠️ CI Automation                          |
+| :--------------------------------------------------------------------------: | :------------------------------------------------------------------------: | :----------------------------------------------------------------: |
+| **[How to setup](./packages/cli/README.md#getting-started)** a basic project | Sort, filter **[your goals](./packages/cli/README.md#portal-integration)** | Updates **[on every PR](./packages/cli/README.md#ci-integration)** |
```

Comment:
Broken link:

```suggestion
| **[How to setup](./packages/cli/README.md#getting-started)** a basic project | Sort, filter **[your goals](./packages/cli/README.md#portal-integration)** | Updates **[on every PR](./packages/cli/README.md#-ci-automation)** |
```

You can test it here: https://github.com/code-pushup/cli/blob/docs-cli-improvements/README.md